config.go

```diff
@@ -685,12 +716,16 @@ func ValidateConfig(cfg Config, usageMessage string,
 		// user has not requested a different location, we'll move the
 		// location to be relative to the specified lnd directory.
 		if cfg.Watchtower.TowerDir == defaultTowerDir {
-			cfg.Watchtower.TowerDir =
-				filepath.Join(cfg.DataDir, defaultTowerSubDirname)
+			cfg.Watchtower.TowerDir = filepath.Join(
+				cfg.DataDir, defaultTowerSubDirname,
+			)
 		}
 	}
 
-	funcName := "loadConfig"
+	funcName := "ValidateConfig"
+	mkErr := func(format string, args ...interface{}) error {
+		return fmt.Errorf(funcName+": "+format, args...)
```

Comment:
Should the `mkErr` return a `usageError` to get it printed to console?

Reply:
No, by default we don't want the usage string (e.g. `Use lnd -h to show usage`) to be printed, only when we know the user provided an incorrect value.